<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNMSettingsDialogBase</class>
 <widget class="QDialog" name="GNMSettingsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove GNM network</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labRemove">
     <property name="text">
      <string>This will remove network from the current project ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkDelete">
     <property name="text">
      <string>Also completely delete network</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labDelete">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="butOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
